CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Changes
 * Implemented LsiModel.docs_processed attribute
 * Added LdaMallet support. Added LdaVowpalWabbit, LdaMallet example to notebook. Added test suite for coherencemodel and aggregation.
   Added `topics` parameter to coherencemodel. Can now provide tokenized topics to calculate coherence value (@dsquareindia, #750)
+* Changed `use_lowercase` option in word2vec accuracy to `case_insensitive` to account for case variations in training vocabulary (@jayantj, #714)
```

Comment:
Please link directly to the github pullrequest/issue page, so people can check the changes and discussions easily.

The markdown syntax for links is `[link text](link url)`.

(@tmylk please start doing this consistently in CHANGELOG starting this release)